.github/stale.yml

```diff
@@ -16,10 +16,10 @@ staleLabel: stale
 
 # Comment to post when marking as stale. Set to `false` to disable
 markComment: >
-  This pull request has been automatically marked as stale because it has not had
-  recent activity. It will be closed in 7 days if no further activity occurs. Please
-  let us know if you are currently working on this PR or have any issues.
-  Thank you for your contributions.
+  Hi @author, I'm going to mark this PR as stale because it hasn't had any updates for 10 days.
```

Comment:
Oh, does this get interpolated? We can refer to the author's name?

Reply:
Actually, we have finally deployed our version of the stale bot: https://github.com/apb7/stale and are now planning to use it (as discussed) [here](https://github.com/oppia/oppiabot/pull/12/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R18) :)

Reply:
Do document that `@author` gets replaced so others know that it's a special keyword here.

Reply:
I have updated the wiki: https://github.com/oppia/oppiabot/wiki
Thanks!

Reply:
and also in the documentation in line 17 :)